SignalServiceKit/src/Devices/OWSGroupsOutputStream.m

```diff
@@ -35,11 +35,6 @@ - (void)writeGroup:(TSGroupModel *)group transaction:(YapDatabaseReadTransaction
         [groupBuilder setAvatarBuilder:avatarBuilder];
     }
 
-    NSData *groupData = [[groupBuilder build] data];
-    uint32_t groupDataLength = (uint32_t)groupData.length;
-    [self.delegateStream writeRawVarint32:groupDataLength];
-    [self.delegateStream writeRawData:groupData];
-
     TSGroupThread *_Nullable groupThread = [TSGroupThread threadWithGroupId:group.groupId transaction:transaction];
     if (groupThread) {
         OWSDisappearingMessagesConfiguration *_Nullable disappearingMessagesConfiguration =
```

Comment:
At this point we'd already serialized the protobuf. Adding a new field at this point is too late.